README.rst

```diff
@@ -8,6 +8,13 @@
 Cirq on IQM
 ###########
 
+**The** ``cirq-iqm`` **package is deprecated and the GitHub repository has been archived. New versions of
+Cirq on IQM will be published as part of the** `iqm-client <https://pypi.org/project/iqm-client/>`_
+**package and can be installed from public PyPI with** ``pip install iqm-client[cirq]``.
+**See the latest documentation at** `<https://docs.meetiqm.com/iqm-client/user_guide_cirq.html>`_ **for more
```

Comment:
Note that this link to `user_guide_cirq.html` will not work until the new `iqm-client` version containing `cirq_iqm` is published.